.github/workflows/docker-image.yml

```diff
@@ -1,26 +1,31 @@
 name: Docker Image CI
 on:
   push:
-    branches: ['master']
+    branches:
+      - master
```

Comment:
do we want to take off the branches here so that we can run this on any updates that are in PR's? or is that handled by the pull_request section?

Reply:
Yes, that's handled by the pull_request section. 👍🏻